hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -61,6 +64,7 @@ public void removeRunningTask(Protos.TaskID taskId) {
     runningTasks.remove(taskId.getValue());
   }
 
```

Comment:
I've spent some time looking to externalize this.  It is some work to do it, it appears.   can we leave it here?   It is somewhat of a flag that says "Fix ME".  I promise to refactor this first on the next PR.  The refactor of code will eliminate the need for the Suppression.